lib/patchelf/patcher.rb

```diff
@@ -184,8 +200,8 @@ def needed_
     end
 
     # @return [String?]
-    def runpath_
-      tag_name_or_log(@rpath_sym, "Entry DT_#{@rpath_sym.to_s.upcase} not found.")
+    def runpath_(rpath_sym = :runpath)
+      tag_name_or_log(rpath_sym, "Entry DT_#{rpath_sym.to_s.upcase} not found.")
```

Comment:
When `RPATH` (or `RUNPATH`) does not exist in the ELF file, for our purposes using `patchelf` as an ELF API, we'd prefer it to silently return `nil` rather than logging.

Reply:
@rmNULL Could you open a PR to move the logging from this method up to the command line? That is to say, maintain the same logging behaviour for the `patchelf.rb` executable, but change this method to return `nil` rather than log.